device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/RetryDecision.java

```diff
@@ -26,6 +29,13 @@ public RetryDecision(boolean shouldRetry, long duration)
         // Codes_SRS_RETRYDECISION_28_001: [The constructor shall save the duration and getRetryDecision]
         this.duration = duration;
         this.shouldRetry = shouldRetry;
+
+        // When a retry policy specifies the descision we should log what the new timing was, this will help with backoff debugging
+        if (!shouldRetry){
+            log.info("NOTE: A new instance of {} has been created with retry disabled, the client will not perform any retries.", RetryDecision.class.getName());
+        } else {
+            log.info("NOTE: A new instance of {} has been created with retry enabled, the client will perform retries every {} milliseconds.", RetryDecision.class.getName(), duration);
```

Comment:
> the client will perform retries every {} milliseconds.

This retry decision is only for one retry, to is should be "the client will retry after {} milliseconds"